build/grunt.js

```diff
@@ -127,6 +127,15 @@ module.exports = function(grunt) {
     },
     dist: {},
     watch: {
+      dist: {
+        files: [
```

Comment:
these are the files that users most likely try to get from the dist folder, so, we copy on changes to them.